tests/storage/business/test_importer_service.py

```diff
@@ -40,16 +40,16 @@ def test_upload_matrix(tmp_path: Path, storage_service_builder) -> None:
         study_factory=Mock(),
     )
 
-    study_url = study_uuid + "/"
     matrix_path = "WRONG_MATRIX_PATH"
-    md = RawStudy(id=study_uuid, workspace=DEFAULT_WORKSPACE_NAME)
+    md = RawStudy(
+        id=study_uuid, workspace=DEFAULT_WORKSPACE_NAME, path=study_path
```

Comment:
No, not anymore. Since the watcher can pick up studies nested in directory. So the upload_matrix (that can be applied onto studies in watched workspaces) must know the path.

Reply:
👍